README.md

```diff
@@ -1,79 +1,69 @@
 # It Is Always Soup
 
-The final team project for Makers.
-A tool to help a user find recipes from leftovers in the fridge.
+## Introduction
+The final team project for Makers Academy. A team of 3 were given just under 2 weeks to go from a vague brief to a full production application.
 
-Our goal - make sure its **not** always soup
+**Our brief** - an eco themed application. <br>
+**Our solution** - a tool to help a user find recipes from their kitchen contents.<br>
+**Our goal** - to make sure it is **not** always soup.
 
-
-------------------
+You can experience the app for yourself here - [It Is Always Soup](https://it-is-always-soup.firebaseapp.com)
 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A video of our final presentation can be seen [here](https://youtu.be/ijyHTx_QyWU)
 
-## Available Scripts
+------------------
+## Technology
+
+We used the following technologies:<br>
```

Comment:
We could also mention our back end server that handled our API requests. I think that's a great talking point and worth mentioning.

Reply:
👍